starter-code/index.html

```diff
@@ -10,5 +10,43 @@
   <title>Ironhack cart</title>
 </head>
 <body>
+      <div class="container">
+
+          <h1 class="tittle">
+              <p>Hironhack</p>
+              <p>Merchandising Shop</p>
+          </h1>
+
+          <div class="market">
+
+              <div class="product">
+
+                  <div class="name-product"><span >Nom del producte</span>
+                  <div id="price-product" class="price-product"><span >Preu unitat del producte</span>
+
+                  <div class ="quantity-product">
+                      <label class="quantity">QTY</label>
+                      <input id="quantity-product" class="quantity" type="number" name="quantity" value="0">
```

Comment:
Little tip, if you don't want to allow negative values, do:
```
<input id="quantity-product" class="quantity" type="number" min="0" name="quantity" value="0">
```
Keep in mind this is not a perfect solution.